packages/service-library/src/models/on-demand-page-scan-result-model.ts

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
```

Comment:
i have just wrapped the code in to a class & added a new method deleteAllDocuments